webservices/common/models.py

```diff
@@ -263,6 +263,16 @@ class CommitteeReports(BaseModel):
     offsets_to_operating_expenditures_period = db.Column(db.Integer)
     offsets_to_operating_expenditures_ytd = db.Column(db.Integer)
 
+    @property
+    def pdf_url(self):
+        if self.report_year is not None and self.report_year > 1994:
```

Comment:
There are at least a handful of reports from 1994 that have PDFs on the FEC site (here's one: http://docquery.fec.gov/pdf/626/95039632626/95039632626.pdf). Should this be `>= 1994`, or does FEC have only partial PDF coverage before 1994?